map.html

```diff
@@ -28,19 +28,19 @@
   <script src="//d3js.org/d3-color.v1.min.js" type="text/javascript"></script>
   <script src="//d3js.org/d3-interpolate.v1.min.js" type="text/javascript"></script>
   <script src="//d3js.org/d3-scale-chromatic.v1.min.js" type="text/javascript"></script>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.3/moment.js" type="text/javascript"></script>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js" type="text/javascript"></script>
+  <!-- <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.3/moment.js" type="text/javascript"></script> -->
```

Comment:
Eventually we will need to make these into bower/yarn/npm/whatever dependencies.